packages/core/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -0,0 +1,174 @@
+                {label && (
+                  <h5
+                    className="ds-u-margin--0 ds-u-padding--1 ds-u-padding-left--2"
+                    id={this.labelId}
+                  >
+                    {label}
+                  </h5>
+                )}
+
+                <ul
+                  aria-labelledby={label ? this.labelId : null}
+                  className="ds-c-list--bare"
+                  role="listbox"
+                >
+                  {items
+                    .filter(
+                      item =>
+                        !inputValue ||
+                        item.name
+                          .toLowerCase()
+                          .includes(inputValue.toLowerCase())
+                    )
+                    .map((item, index) => (
+                      <li
+                        aria-selected={highlightedIndex === index}
+                        className={
+                          highlightedIndex === index
+                            ? 'ds-c-autocomplete__list-item ds-c-autocomplete__list-item--active'
+                            : 'ds-c-autocomplete__list-item'
+                        }
+                        key={item.id}
+                        role="option"
+                        {...getItemProps({ item })}
+                      >
+                        {itemToString(item)}
+                      </li>
+                    ))}
+                </ul>
+              </div>
+            ) : null}
+
+            <Button
+              aria-label={clearAriaLabel}
+              className="ds-u-float--right ds-u-padding-right--0"
+              href="javascript:void(0);"
+              onClick={clearSelection}
+              size="small"
+              variation="transparent"
+            >
+              {clearInputText}
+            </Button>
+          </div>
+        )}
+      />
+    );
+  }
+}
+
+Autocomplete.defaultProps = {
+  clearAriaLabel: 'Clear typeahead and search again',
+  clearInputText: 'Clear search'
+};
+
+Autocomplete.propTypes = {
+  children: PropTypes.node,
+  /**
+   * Screenreader-specific label for the Clear input link. Intended to provide a longer, more descriptive explanation of the link's behavior.
+   */
+  clearAriaLabel: PropTypes.string,
+  /**
+   * Clear link text that will appear on the page as part of the rendered component
+   */
+  clearInputText: PropTypes.node,
+  /**
+   * Used to determine the string value for the selected item (which is used to compute the `inputValue`.
+   *
+   * Also see: https://github.com/paypal/downshift#itemtostring
+   */
+  itemToString: PropTypes.func,
+  /**
+   * Array of objects used to populate the suggestion list that appears below the input as users type. This array of objects is intended for an async data callback, and should conform to the prescribed shape to avoid errors.
+   */
+  items: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      name: PropTypes.name
```

Comment:
This isn't a valid `PropType`, I think you meant `string`? Not sure why React isn't throwing an error about this though.

Reply:
@sawyerh The React docs list `PropTypes.shape` as a valid type: https://reactjs.org/docs/typechecking-with-proptypes.html

I felt it could be a good way to get back an object with the necessary attributes. If there's a better approach, I'll switch this up.

```
// An object taking on a particular shape
  optionalObjectWithShape: PropTypes.shape({
    color: PropTypes.string,
    fontSize: PropTypes.number
  })
```

Reply:
@1Copenut Sorry, to clarify: I'm referring to `PropTypes.name` being invalid. Did you mean to have `.string` rather than `.name`?

Reply:
@sawyerh Yes, that was meant to be `.string`. Corrected locally, will be part of the next code push.